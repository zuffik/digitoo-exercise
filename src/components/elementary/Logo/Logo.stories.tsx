import {Logo} from './Logo';

export default {
    title: 'Elementary',
};

export const logo = Logo;
