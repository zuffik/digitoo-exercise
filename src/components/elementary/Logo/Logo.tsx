import * as React from 'react';
import {Assets} from '../../../services/data/Assets';
import styles from './Logo.module.sass';

type Props = React.SVGProps<SVGSVGElement> & {
  title?: string;
};

export const Logo: React.FC<Props> = (props: Props) => {
  const Component = Assets.logo.component;
  return <Component {...props} className={styles.logo} />;
};
