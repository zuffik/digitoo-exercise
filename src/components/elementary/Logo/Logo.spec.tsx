import * as React from 'react';
import {render} from '@testing-library/react'
import {Logo} from "./Logo";

describe('<Logo/>', () => {
    it('should render', () => {
        const {asFragment} = render(<Logo/>);
        expect(asFragment()).toMatchSnapshot();
    });
});
