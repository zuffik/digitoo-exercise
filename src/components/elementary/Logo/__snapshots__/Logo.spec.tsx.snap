// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo/> should render 1`] = `
<DocumentFragment>
  <svg
    class="logo"
  >
    logo.svg
  </svg>
</DocumentFragment>
`;
